zuul.d/jobs.yaml

```diff
@@ -176,3 +176,13 @@
     run: playbooks/terraform-provider-flexibleengine-acceptance-test-orange/run.yaml
     secrets:
       - orange_credentials
+
+# terraform-provider-opentelekomcloud acceptance tests with Telekom cloud
+- job:
+    name: terraform-provider-opentelekomcloud-acceptance-test-telekom
```

Comment:
Please use fullname for the last word.
s/telekom/opentelekomcloud/

Reply:
done